hack/tests/subcommand-olm-install.sh

```diff
@@ -13,11 +13,11 @@ test_version() {
 
     # Status should fail with OLM not installed
     commandoutput=$(operator-sdk olm status 2>&1 || true)
-    echo $commandoutput | grep -F "Failed to get OLM status: no existing installation found"
+    echo $commandoutput | grep -F "Failed to get OLM status"
 
     # Uninstall should fail with OLM not installed
     commandoutput=$(operator-sdk olm uninstall 2>&1 || true)
-    echo $commandoutput | grep -F "Failed to uninstall OLM: no existing installation found"
+    echo $commandoutput | grep -F "Failed to uninstall OLM"
```

Comment:
Since we are logging the error too, checking just whether installation/fetching status failed or not.